### ISSUE TITLE
Create OA3 Spec for Pub Service

### ISSUE THREAD

**Issue by springframeworkguru** [sfg-beer-works/sfg-beer-works#61]
setup repo, spec, and redoc

**Comment by springframeworkguru** [comment]
Created Repo here:

Can use spec from distributor just for setup

**Comment by atkawa7** [comment]
@springframeworkguru Checkout https://github.com/atkawa7/httpsnippet for generating code snippets with springfox and redoc

**Comment by springframeworkguru** [comment]
Springfox doesn't support oa3 yet

**Comment by atkawa7** [comment]
@springframeworkguru True. But you can support things such as x-logo and x-code-sample with the current springfox and redoc using extensions. Checkout the demo https://github.com/atkawa7/httpsnippet
![image](https://user-images.githubusercontent.com/5673828/54004150-35176380-415d-11e9-9ca7-fec527131fa1.png)
